lib/charms/layer/openstack.py

```diff
@@ -702,9 +702,11 @@ def create_sg_rule(self, sg_id, address, port):
                    sg_id)
 
     def get_port_sec_enabled(self):
-        subnet_info = _openstack('subnet', 'show', self.subnet)
-        network_info = _openstack('network', 'show', subnet_info['network_id'])
-        return network_info['port_security_enabled']
+        if self.subnet:
```

Comment:
`get_por_sec_enabled` is expected to return True or False. Returning `NoneType` would have the same effect, but I would suggest returning False.

```python
if not self.subnet:
    return False
# the 3 statements as initially written, without indentation
```